source/_components/light.hyperion.markdown

```diff
@@ -33,3 +33,13 @@ Configuration variables:
 - **hdmi_priority** (*Optional*): The priority of the HDMI grabber of this Hyperion instance, note that this priority must be higher than all other priorities used for correct behavior. Defaults to `880`.
 - **default_color** (*Optional*): The color of the light. Defaults to `[255, 255, 255]`.
 - **effect_list** (*Optional*): The list of effects that can be used. Defaults to `['HDMI', 'Cinema brighten lights', 'Cinema dim lights', 'Knight rider', 'Blue mood blobs', 'Cold mood blobs', 'Full color mood blobs', 'Green mood blobs', 'Red mood blobs', 'Warm mood blobs', 'Police Lights Single', 'Police Lights Solid', 'Rainbow mood', 'Rainbow swirl fast', 'Rainbow swirl', 'Random', 'Running dots', 'System Shutdown', 'Snake', 'Sparks Color', 'Sparks', 'Strobe blue', 'Strobe Raspbmc', 'Strobe white', 'Color traces', 'UDP multicast listener', 'UDP listener', 'X-Mas']`.
+
+A few notes:
+
+- When an effect is called with the light.turn_on service, aditional arguments to the effect can be specified using the following syntax: 
```

Comment:
I don't wanna be picky about something...
But you sate `A few notes:`, is followed by a list containing one single note?

Reply:
Also `aditional ` -> `additional`

Reply:
Thanks for your feedback!
I made some changes and added some aditional information.
Could you have a second look at it?